.changeset/tender-walls-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": patch
+---
+
+fix(OnyxInput, OnyxTextarea): Adjust selected text color
```

Comment:
Please use a descriptive text that lets people understand the changes without additional context:
```suggestion
fix(OnyxInput, OnyxTextarea): Adjust selection highlighting for better contrast
```

I tend to reuse the text from the changeset in the PR description, that way you're don't need to write it twice :)